### PR TITLE
Added HW10

### PR DESCRIPTION
**Pull request by YuriiYevpak**

### CODE REVIEW

**Comment by YuriiYevpak** [comment]
Не певен по тасці 3

File: YuriiYevpak/HW10/YuriiYevpakHW10_T1.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# # Таска 1
+class Polygon:
```
Comment (kolyasalubov) [inline YuriiYevpak/HW10/YuriiYevpakHW10_T1.py:2]: good job

File: YuriiYevpak/HW10/YuriiYevpakHW10_T1.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+
+# Приклад використання:
+rectangle = Rectangle(length=5, width=4)
+print("Площа прямокутника:", rectangle.area())  # Вивід: 20
+
+
+
+# # Таска 2
+class Human:
```
Comment (kolyasalubov) [inline YuriiYevpak/HW10/YuriiYevpakHW10_T1.py:27]: good job

File: YuriiYevpak/HW10/YuriiYevpakHW10_T1.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+for name in names:
+    human = Human(name)
+
+
+print(human.welcome_message())  # Вивід: Ласкаво просимо, {name}!
+
+print(Human.species_info())  # Вивід: Це вид Homosapiens.
+
+print(Human.arbitrary_message())  # Вивід: Це статичний метод. 
+
```
Comment (kolyasalubov) [inline YuriiYevpak/HW10/YuriiYevpakHW10_T1.py:55]: thanks

File: YuriiYevpak/HW10/YuriiYevpakHW10_T1.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+# Введення імен з терміналу
+names = input("Введіть імена людей через кому: ").split(", ")
+
+# Створення екземплярів класу Human для кожного введеного імені
+for name in names:
+    human = Human(name)
+    print(human.welcome_message())  # Вивід: Ласкаво просимо, {ім'я}!
+
+
+
```
Comment (kolyasalubov) [inline YuriiYevpak/HW10/YuriiYevpakHW10_T1.py:66]: good job

File: YuriiYevpak/HW10/YuriiYevpakHW10_codewars.py [hunk 5]
```diff
@@ -0,0 +1,158 @@
+# # 1 Ball_syper_ball
```
Comment (kolyasalubov) [inline YuriiYevpak/HW10/YuriiYevpakHW10_codewars.py:1]: excellent work

**Review by kolyasalubov — APPROVED** [review]

**Pull request merged**